### ISSUE TITLE
Result from pretrain weight is inferior than the one in the report 

### ISSUE THREAD

**Issue by burin-n** [FactoDeepLearning/VerticalAttentionOCR#2]
I test iam_line.pt and got 3.52% CER on validation set. 
Could you verify that the weight is the right one?

**Comment by FactoDeepLearning** [comment]
The weights are good. As mentioned in the preprint, we used a mini-batch size of 16. I've updated the main file.
